doc/radosgw/elastic-sync-module.rst

```diff
@@ -4,6 +4,9 @@ ElasticSearch Sync Module
 
 .. versionadded:: Kraken
 
+.. note::
+     As of 31 May 2020, only Elasticsearch 6 and lower are supported. ElasticSearch 7 is not supported.
```

Comment:
```suggestion
     ElasticSearch 7 is not supported. As of 31 May 2020, only Elasticsearch 6 and lower are supported.

Reply:
Personally, I would flip this. The important information is that ElasticSearch 7 is not supported, so that's key info straight away. After that, the explanation of what you can have. But it's much of a muchness.

Reply:
I disagree.